src/defaultOptions.js

```diff
@@ -29,6 +29,10 @@ const everyMonthOptions = {
         amount: '100',
         bgColor: '#A0CBFE',
       },
+      {
+        amount: '500',
+        bgColor: '#A0CBFE',
```

Comment:
Is this `bgColor` used, could we remove it?

Reply:
No, we are not using them after #29. I'll remove them